lib/graphql/types/json.rb

```diff
@@ -10,6 +10,11 @@ module Types
     #
     #     argument :template_parameters, GraphQL::Types::JSON, null: false
     #
+    # Note that when the JSON scalar is used an argument for subscriptions, its value will be used to construct
+    # a topic name. You are responsible to ensure that any values provided by the client and the server are
+    # ordered identically such that both the client and the server agree on the correct topic name.
```

Comment:
This doesn't _seem_ right to me... 

Given an operation like 

```graphql 
subscription($filter: JSON!) {
  tweetWasTweeted(filter: $filter) { ... }
}
```

I would _expect_ that `filter: { handle: "matz", hashtags: ["#ruby"] }` would subscribe to the same events as `filter: { hashtags: ["#ruby"], handle: "matz" }`. (Arguments in GraphQL are _not_ ordered, anyways.)

Is there a reason to _prefer_ order-sensitive JSON inputs? If not, do you mind trying to normalize them on the server?